enclave/core/CMakeLists.txt

```diff
@@ -46,6 +48,9 @@ add_library(oecore STATIC
     ${PLATFORM_SRC}
     )
 
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+add_compile_options(-Wno-error=unused-command-line-argument)
```

Comment:
What command line option was added that requires this suppression for Clang, here and in tests/crypto/host/CMakeLists.txt?

Reply:
These lines are now not required removed